src/handlers/events/me.ts

```diff
@@ -0,0 +1,299 @@
+import * as _ from 'lodash';
+import * as db from '../../database';
+import { formatDate } from '../../utils';
+import type { Context } from '../../types/telegraf';
+import logger from '../../logger';
+import * as messages from '../../constants/messages';
+import { sendExpl } from '../../utils';
+import { User } from 'telegraf/typings/telegram-types';
+
+const DATA_SEPARATOR = '|';
+const EXPLS_IN_PAGE = 5;
+
+function getListTitle(page: string) {
+  switch (page) {
+    case 'latest':
+      return messages.me.latestByYou();
+    case 'oldest':
+      return messages.me.oldestByYou();
+    case 'best':
+      return messages.me.bestByYou();
+    case 'worst':
+      return messages.me.worstByYou();
+    case 'likes':
+      return messages.me.likedByYou();
+    default:
+      return '';
+  }
+}
+
+function button(text: string, data: string | string[]) {
+  if (_.isString(data)) {
+    data = [data];
```

Comment:
[_.castArray](https://lodash.com/docs/4.17.15#castArray) achieves this without the need of if-clause